transitive_deps.bzl

```diff
@@ -6,4 +6,4 @@ load("@rules_rust//rust:repositories.bzl", "rust_repositories")
 def cargo_raze_transitive_deps():
     """Loads all dependnecies from repositories required for cargo-raze"""
     rules_foreign_cc_dependencies()
-    rust_repositories(include_rustc_srcs = True)
+    rust_repositories(edition = "2018", include_rustc_srcs = True)
```

Comment:
Can `include_rustc_srcs` be removed since rust_analyzer is being removed?

Reply:
This is still needed in one way or another, aiui. rust_analyzer still works, it's just an aspect now, and doesn't need an explicit rule. The workspace dependency on it is still there. We've gone back and forth on the `.bazelrc` approach vs this one, I just left it.

See: https://bazelbuild.github.io/rules_rust/rust_analyzer.html

Reply:
Sounds good, thanks for the clarification.